src/scripts/mdExts.js

```diff
@@ -0,0 +1,3953 @@
+"use strict";

```

Comment:
This is spectacular. You should consider refactoring this into a standalone winmd reader library for JavaScript. 😉